components/brave_sync/brave_sync_prefs.h

```diff
@@ -25,6 +25,9 @@ class PrefRegistrySyncable;
 
 namespace brave_sync {
 
+extern const char kSyncV2MigrateNoticeDismissed[];
```

Comment:
Can't we use JNI to access these two prefs? They are all hidden intentionally and can only be accessed by Prefs methods.

Reply:
Yes, we can,  if this is vital.

Thanks for the notice, will update.

Reply:
@darkdh 
fixed at https://github.com/brave/brave-core/pull/6204/commits/dc9d77acf84584be876acc25641cab787139eaf5